app/src/main/java/com/github/polypoly/app/menu/kotlin/GameMusic.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.polypoly.app.menu.kotlin
+
+import android.content.Context
+import android.media.MediaPlayer
+import java.lang.Float.max
+import kotlin.math.min
+
+class GameMusic(private val context: Context, private val songId: Int) {
+    private lateinit var mediaPlayer: MediaPlayer
+
+    fun startSong() {
+        mediaPlayer = MediaPlayer.create(context, songId)
+        mediaPlayer.isLooping = true
+        mediaPlayer.start()
+    }
+
+    fun setVolume(value: Float) {
+        var validValue = max(value, 1f)
+        validValue = min(validValue, 0f)
```

Comment:
You can use `clamp` to bind validValue between 0f ans 1f

Reply:
Nice! I didn't know about this, thanks :)

Reply:
No problem :)